cmd/influx_inspect/reportdisk/reportdisk_test.go

```diff
@@ -0,0 +1,3 @@
+package reportdisk_test
+
+// TODO: write some tests
```

Comment:
Copied from `report_test.go` - I've tested this manually but open to writing a test. Would probably have to commit a tar'd shard directory to test on.